configure.ac

```diff
@@ -1118,7 +1118,7 @@ M0_CPPFLAGS_DIST=$(echo "$M0_CPPFLAGS" | grep -Po -e '-D\S+|-I/usr\S+|-inc\S+' |
 # __attribute__, such as __attribute__((gccxml(...))). It's important because we
 # use -Werror, which turns warnings into errors, but we still need to be able to
 # use gccxml attributes.
-M0_CFLAGS="-pipe -Wall -Werror -Wno-attributes -fno-strict-aliasing $M0_CFLAGS"
+M0_CFLAGS="-pipe -Wall -Werror -Wno-array-bounds -Wno-return-local-addr -Wno-deprecated-declarations -Wno-attributes -fno-strict-aliasing $M0_CFLAGS"
```

Comment:
The regression job is failing due to -Wno-array-bounds option. Also please check Hua's comment.

Reply:
ack.